<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#LoginButton:hover{
text-decoration: underline;
font-weight:bold;
cursor: pointer;
}
QLineEdit#Username:focus{
border:none;
border-bottom: 1.5px solid #4A8292;
}

QLineEdit.password:active{
border:none;
border-bottom: 1.5px solid #4A8292;
}

QPushButton#SingUpBtn:hover{
border:none;
border-bottom: 1.5px solid #628D99;}
QPushButton#SingUpBtn_3:hover{
border:none;
border-bottom: 1.5px solid #628D99
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="mainWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1366</width>
      <height>768</height>
     </rect>
    </property>
    <widget class="QWidget" name="Login">
     <property name="styleSheet">
      <string notr="true">background-color:#97BDC8</string>
     </property>
     <widget class="QStackedWidget" name="loginSTKWidgetForm">
      <property name="geometry">
       <rect>
        <x>255</x>
        <y>171</y>
        <width>856</width>
        <height>425</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
border-radius:30px;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>-1</y>
          <width>428</width>
          <height>425</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/img/img/background login.png);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>281</width>
           <height>81</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>19</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#454545;
</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="plainText">
          <string>Brain Storm
Co-working Space</string>
         </property>
        </widget>
       </widget>
       <widget class="QStackedWidget" name="loginForm">
        <property name="geometry">
         <rect>
          <x>426</x>
          <y>0</y>
          <width>431</width>
          <height>425</height>
         </rect>
        </property>
        <widget class="QWidget" name="page_4">
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>137</x>
            <y>370</y>
            <width>131</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Create new account?</string>
          </property>
         </widget>
         <widget class="QPushButton" name="LoginButton">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>310</y>
            <width>75</width>
            <height>31</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;
border-radius:8px;
background-color: #4A8292;


</string>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>40</y>
            <width>161</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-weight:bold;
color:#454545;</string>
          </property>
          <property name="text">
           <string>Sing In</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="Username">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>170</y>
            <width>341</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;
border-bottom: 1.5px solid #d9d9d9;
</string>
          </property>
          <property name="placeholderText">
           <string>Username</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="password">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>240</y>
            <width>341</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;
border-bottom: 1.5px solid #d9d9d9;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Password</string>
          </property>
         </widget>
         <widget class="QPushButton" name="SingUpBtn">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>366</y>
            <width>51</width>
            <height>24</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#628D99;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Sing Up</string>
          </property>
         </widget>
         <widget class="QLabel" name="usernameIconeGrey">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>170</y>
            <width>17</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">width:16px;
  
height: 16px;  </string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/img/img/alternate_email_black_24dp.png</pixmap>
          </property>
         </widget>
         <widget class="QPushButton" name="visibilityBtn">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>240</y>
            <width>16</width>
            <height>16</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(&quot;C:/Users/Mon Pc/Desktop/Qt projects/BrainStormSpace/img/visibilityOff.png&quot;);
background-repeat: no-repeat;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox">
          <property name="geometry">
           <rect>
            <x>315</x>
            <y>314</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <zorder>checkBox</zorder>
         <zorder>label</zorder>
         <zorder>LoginButton</zorder>
         <zorder>label_3</zorder>
         <zorder>Username</zorder>
         <zorder>password</zorder>
         <zorder>SingUpBtn</zorder>
         <zorder>usernameIconeGrey</zorder>
         <zorder>visibilityBtn</zorder>
        </widget>
        <widget class="QWidget" name="page_5">
         <widget class="QLineEdit" name="Username_SingUp">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>130</y>
            <width>341</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;
border-bottom: 1.5px solid #d9d9d9;
</string>
          </property>
          <property name="placeholderText">
           <string>Username</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>40</y>
            <width>161</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-weight:bold;
color:#454545;</string>
          </property>
          <property name="text">
           <string>Sing UP</string>
          </property>
         </widget>
         <widget class="QPushButton" name="SingUpBtn_2">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>340</y>
            <width>75</width>
            <height>31</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;
border-radius:8px;
background-color: #4A8292;


</string>
          </property>
          <property name="text">
           <string>Sign Up</string>
          </property>
         </widget>
         <widget class="QPushButton" name="visibilityBtn_SingUp">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>180</y>
            <width>16</width>
            <height>16</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(&quot;C:/Users/Mon Pc/Desktop/Qt projects/BrainStormSpace/img/visibilityOff.png&quot;);
background-repeat: no-repeat;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLineEdit" name="password_SingUp">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>180</y>
            <width>341</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;
border-bottom: 1.5px solid #d9d9d9;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Password</string>
          </property>
         </widget>
         <widget class="QLabel" name="usernameIcone_SingUp">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>130</y>
            <width>17</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">width:16px;
  
height: 16px;  </string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/img/img/alternate_email_black_24dp.png</pixmap>
          </property>
         </widget>
         <widget class="QLineEdit" name="email_SingUp">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>230</y>
            <width>341</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;
border-bottom: 1.5px solid #d9d9d9;
</string>
          </property>
          <property name="placeholderText">
           <string>Email</string>
          </property>
         </widget>
         <widget class="QLabel" name="emailIcone_SingUp">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>230</y>
            <width>17</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">width:16px;
  
height: 16px;  </string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/img/img/email_grey.png</pixmap>
          </property>
         </widget>
         <widget class="QLabel" name="numberIcone_SingUp">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>280</y>
            <width>17</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">width:16px;
  
height: 16px;  </string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/img/img/phone_grey.png</pixmap>
          </property>
         </widget>
         <widget class="QLineEdit" name="number_SingUp">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>280</y>
            <width>341</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;
border-bottom: 1.5px solid #d9d9d9;
</string>
          </property>
          <property name="placeholderText">
           <string>Phone Number</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_SingUp">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>340</y>
            <width>21</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>380</y>
            <width>221</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Do you have allready an account ?</string>
          </property>
         </widget>
         <widget class="QPushButton" name="SingUpBtn_3">
          <property name="geometry">
           <rect>
            <x>223</x>
            <y>375</y>
            <width>51</width>
            <height>24</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#628D99;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Sign In</string>
          </property>
         </widget>
         <zorder>checkBox_SingUp</zorder>
         <zorder>Username_SingUp</zorder>
         <zorder>label_4</zorder>
         <zorder>SingUpBtn_2</zorder>
         <zorder>password_SingUp</zorder>
         <zorder>usernameIcone_SingUp</zorder>
         <zorder>visibilityBtn_SingUp</zorder>
         <zorder>email_SingUp</zorder>
         <zorder>emailIcone_SingUp</zorder>
         <zorder>number_SingUp</zorder>
         <zorder>numberIcone_SingUp</zorder>
         <zorder>label_2</zorder>
         <zorder>SingUpBtn_3</zorder>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3"/>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
